app/lib/admin/actions/create_publisher.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:clock/clock.dart';
+
+import 'package:pub_dev/account/backend.dart';
+import 'package:pub_dev/admin/actions/actions.dart';
+import 'package:pub_dev/audit/models.dart';
+import 'package:pub_dev/publisher/models.dart';
+import 'package:pub_dev/shared/datastore.dart';
+
+final createPublisher = AdminAction(
+    name: 'create-publisher',
+    options: {
+      'publisher': 'name of publisher to create',
+      'member': 'email of user to add',
```

Comment:
maybe call it `--member-email` so we can also use `member-user-id` later if needed?